packages/useWorker/src/useWorker.ts

```diff
@@ -166,6 +166,6 @@ export const useWorker = <T extends (...fnArgs: any[]) => any>(
   }, [killWorker])
 
   return [
-    workerHook, workerController,
```

Comment:
unlike a state ( "useState" ), a ref, does not trigger a re-render, so even if we change "worker.current" whoever integrates the useWorker hook will probably not have access to the latest worker.current ref... but maybe I'm wrong, did you have a moment to check it?